controllers/managedocs_controller.go

```diff
@@ -153,6 +149,7 @@ type ManagedOCSReconciler struct {
 	addonParams                        map[string]string
 	onboardingValidationKeySecret      *corev1.Secret
 	servingCertCaBundleConfigMap       *corev1.ConfigMap
+	kubeRBACConfigMap                  *corev1.ConfigMap
```

Comment:
This is not a general kube rbac config map, it is specifically for prometheus
```suggestion
	prometheusKubeRBACConfigMap                  *corev1.ConfigMap
```